pkg/httpd/sftpvolume.go

```diff
@@ -269,7 +269,7 @@ func (u *UserVolume) UploadFile(dirPath, uploadPath, filename string, reader io.
 		return rest, fmt.Errorf("the provided reader does not implement io.ReaderAt")
 	}
 
-	if err1 := u.recorder.ChunkedRecord(fd.FTPLog, readerAt, 0, totalSize); err1 != nil {
+	if err1 := u.recorder.Record(fd.FTPLog, reader); err1 != nil {
 		logger.Errorf("Record file err: %s", err1)
 	}
 
```

Comment:
The code has no issues to mention and does not require any change. It is already correct with no potential irregularities or unnecessary changes. The provided code functions without any mistakes.

Please let me know if you need further assistance!